discrete_optimization/generic_tools/callbacks/loggers.py

```diff
@@ -49,3 +49,40 @@ def on_step_end(
         logger.log(
             msg=f"Iteration #{self.nb_iteration}", level=self.step_verbosity_level
         )
+
+
+class ObjectiveLogger(Callback):
+    """
```

Comment:
Maybe the docstrings should be changed ;p

Reply:
well seen, actually same for the already existing NbIterationLogger right ?!

Reply:
... correct.

Reply:
Done